xmtp_id/src/associations/member.rs

```diff
@@ -78,18 +78,21 @@ pub struct Member {
     pub identifier: MemberIdentifier,
     pub added_by_entity: Option<MemberIdentifier>,
     pub client_timestamp_ns: Option<u64>,
+    pub added_on_chain_id: Option<u64>,
```

Comment:
Just double-checking, do we have a constrained number of chains? E.g., are we protected against someone producing a SCW signature for an address they don't own on a bogus chain? And do revocation signatures have to match the chain on which a SCW was added?

Reply:
> do we have a constrained number of chains

Yes. We have a fixed list of chains and to add more we will have to roll the update out to everyone and deprecate old clients.

> And do revocation signatures have to match the chain on which a SCW was added

That PR should be up shortly